File: task-03/prime.js
function is_prime(num){
    if (num < 2){
        return false;
    }
    for(let i=2;i<= Math.sqrt(num);i++){
        if(num%i==0){
            return false;
        }
    }
    return true;
}

function main(){
    const readline=require('readline').createInterface({
        input: process.stdin,
        output: process.stdout
    });

    readline.question("Enter the limit n: ", function(limit){
        const n=parseInt(limit);
        console.log();

        for(let i=0;i<n;i++){
            if(is_prime(i)){
                process.stdout.write(i+" ");
            }
        }

        console.log();
        readline.close();
    });


}

main();

File: task-03/prime.py
def is_prime(num):
    if num < 2:
        return false
    for i in range(2,int (num ** 0.5)+1):
        if num%i==0:
            return False
        return True

    def main():
        n=int(input("Enter the limit n:"))
        print()

        for i in range(2,n+1):
            if is_prime(i):
                print(i,end=" ")

    if __name__ == "__main__":
        main()
